runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -131,14 +131,21 @@ public Layout getCurrentLayout() {
             } catch (InterruptedException ie) {
                 throw new UnrecoverableCorfuInterruptedError("Interrupted in layoutHelper", ie);
             } catch (ExecutionException ex) {
-                log.warn("Error executing remote call, invalidating view and retrying in {} ms",
-                        retryRate, ex);
-
-                // If SystemUnavailable exception is thrown by the layout.get() completable future,
-                // the exception will materialize as an ExecutionException. In that case, we need to propagate
-                // this exception.
-                if (ex.getCause() instanceof SystemUnavailableError) {
-                    throw (SystemUnavailableError) ex.getCause();
+                log.warn("layoutHelper: Error executing remote call, invalidating view and "
```

Comment:
Maybe move this log to after the two if clauses. (as if we encounter Error or RuntimeException, there won't be any invalidating and retrying)

Reply:
Done.

Reply:
Thanks